Formula/gammaray.rb

```diff
@@ -31,6 +31,7 @@ def install
     ENV.libcxx
 
     system "cmake", *std_cmake_args,
+                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
```

Comment:
This may not actually be appropriate here. But the directory structure is a bit complicated, will need to look at it more carefully.

Reply:
All other libraries under `GammaRay.app/Contents/PlugIns/gammaray/2.11/qt5_15-arm64` do have rpaths.

Could try modifying corresponding CMakeLists.txt to use upstream's helper function. I think it would be the following, but will need to confirm:
```diff
diff --git a/plugins/positioning/CMakeLists.txt b/plugins/positioning/CMakeLists.txt
index 6de29cbc6..8b9b61c4c 100644
--- a/plugins/positioning/CMakeLists.txt
+++ b/plugins/positioning/CMakeLists.txt
@@ -17,6 +17,7 @@ if (NOT QtCore_VERSION VERSION_LESS 5.7)
     )

     add_library(gammaray_geopositioninfosource MODULE ${gammaray_geopositioninfosource_srcs})
+    gammaray_set_rpath(gammaray_geopositioninfosource ${TARGET_PLUGIN_INSTALL_DIR}/position)
     target_compile_features(gammaray_geopositioninfosource PUBLIC ${GAMMARAY_REQUIRED_CXX_FEATURES})
     target_link_libraries(gammaray_geopositioninfosource
         gammaray_common
```